lib/cisco_node_utils/version.rb

```diff
@@ -14,5 +14,5 @@
 
 # Container module for version number only.
 module CiscoNodeUtils
-  VERSION = '1.0.2-dev'
+  VERSION = '1.0.2'
```

Comment:
Do you get the same build error from #9?  Run ```gem --version``` and ensure that it is version 2.1 or higher